contracts/bprotocol/BComptroller.sol

```diff
@@ -37,9 +32,9 @@ contract BComptroller {
         bool is_cETH = cToken == registry.cEther();
         address bToken;
         if(is_cETH) {
-            bToken = address(new BEther(address(registry), cToken, pool));
+            bToken = address(new BEther(address(registry), cToken, registry.pool()));
```

Comment:
see comment below, but in general this should not be a c'tor param, as it will be upgradbale later.